dipy/reconst/shm.py

```diff
@@ -1575,3 +1576,95 @@ def convert_sh_to_legacy(sh_coeffs, sh_basis, full_basis=False):
         raise ValueError("Invalid basis name.")
 
     return out_sh_coeffs
+
+
+def _setup_wigner_rotation(b, alpha, beta, gamma):
```

Comment:
alpha, beta, gamma could be 1 argument. Why did you choose to split it ?

Reply:
They can be converted into a list but these variables are user inputs so for better readability and understanding they are separate. Should I convert them to a list?